dev/how-to/deploy/data-migration-with-ansible.md

```diff
@@ -349,15 +349,15 @@ dm-worker2 ansible_host=172.16.10.73 source_id="mysql-replica-02" server_id=102
 
 > **Note:**
 >
-> If `relay_binlog_name` is not set, DM-worker pulls the binlog starting from the earliest existing binlog file of the upstream MySQL or MariaDB. In this case, it can take a long period of time to pull the latest binlog for the data synchronization task.
+> If `relay_binlog_name` is not set, DM-worker pulls the binlog starting from the earliest existing binlog file of the upstream MySQL or MariaDB. In this case, it can take a long period of time to pull the latest binlog for the data replication task.
```

Comment:
It is a bit ambiguous here, it is not going to take a long time to pull the latest file. It is going to retrieve all the files and that is why it takes so long.
```suggestion
> If `relay_binlog_name` is not set, DM-worker pulls the binlog starting from the earliest existing binlog file of the upstream MySQL or MariaDB. In this event, it may take a significant amount of time to retrieve all of the binlog files.
```